lib/rubocop/cop/mixin/comments_help.rb

```diff
@@ -75,6 +77,14 @@ def find_end_line(node)
         end
       end
       # rubocop:enable Metrics/AbcSize
+
+      def if_with_else?(node)
+        node.if_type? && node.else? && node.loc.else
```

Comment:
FYI, `node.loc.else` looks unnecessary. That's why it's removed in the above suggestion.